base/shell/explorer/lang/pt-PT.rc

```diff
@@ -7,7 +7,7 @@
  *              Copyright 2021 Gabriel Aguiar <fgsoftwarestudio@gmail.com>
  */
 
-LANGUAGE LANG_PORTUGUESE, SUBLANG_NEUTRAL
+LANGUAGE LANG_PORTUGUESE, SUBLANG_PORTUGUESE
```

Comment:
The majority of files has `SUBLANG_NEUTRAL`, is this something that recently changed?

Reply:
I have mistakenly defined SUBLANG_NEUTRAL. After reading more carefully the text at https://reactos.org/wiki/Sublang, I see that SUBLANG_PORTUGUESE is the correct one when there is a translation for SUBLANG_PORTUGUESE_BRAZILIEN, and SUBLANG_NEUTRAL when there is no translation for Brazilian.